Client/Frontend/Browser/TabManager.swift

```diff
@@ -67,6 +67,7 @@ class TabManager : NSObject {
         let configuration = WKWebViewConfiguration()
         configuration.processPool = WKProcessPool()
         configuration.preferences.javaScriptCanOpenWindowsAutomatically = !(self.prefs.boolForKey("blockPopups") ?? true)
+        configuration.mediaPlaybackRequiresUserAction = !(self.prefs.boolForKey("allowAutoplay") ?? true)
```

Comment:
actually, [`mediaPlaybackRequiresUserAction`](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIWebView_Class/#//apple_ref/occ/instp/UIWebView/mediaPlaybackRequiresUserAction) and [`allowsInlineMediaPlayback`](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIWebView_Class/#//apple_ref/occ/instp/UIWebView/allowsInlineMediaPlayback) are different properties - both have been available since iOS 4.

[`requiresUserActionForMediaPlayback`](https://developer.apple.com/library/ios/documentation/WebKit/Reference/WKWebViewConfiguration_Ref/index.html#//apple_ref/occ/instp/WKWebViewConfiguration/requiresUserActionForMediaPlayback) is the property that replaced `mediaPlaybackRequiresUserAction` in iOS 9.0.

these names are so long and confusing, heh. anyway, fixing. thanks!